pkg/ddl/create_table.go

```diff
@@ -1003,6 +1003,8 @@ func setEmptyConstraintName(namesMap map[string]bool, constr *ast.Constraint) {
 				switch constr.Tp {
 				case ast.ConstraintVector:
 					colName = getAnonymousIndexPrefix(model.ColumnarIndexTypeVector)
+				case ast.ConstraintColumnar:
```

Comment:
Should we consider renaming this index to `ConstraintInverted or a more specific name`? It seems that both `ColumnarIndexTypeInverted` and `ColumnarIndexTypeVector` should be categorized as ConstraintColumnar.